src/components/common/ui/breadcrumbs-mui/Breadcrumbs.styles.ts

```diff
@@ -0,0 +1,37 @@
+import { SxProps, Theme } from '@mui/system';
+
+import palette from '@/styles/theme/constants/pallete/palette';
+
+export const homeIcon: SxProps<Theme> = {
+  width: '14px',
+  height: '14px',
+  transform: 'translateY(-1px)',
+};
+
+export const breadcrumb: SxProps<Theme> = {
+  li: {
+    '&:last-child': {
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      whiteSpace: 'nowrap',
+      flex: 1,
+    },
+  },
+  a: {
+    gap: '4px',
+    display: 'flex',
+    alignItems: 'center',
+    color: palette.grey[700],
+    span: {
```

Comment:
Since you removed span from component, your styles for span doesn't work and you don't have ellipsis when the screen is getting smaller. 
Fix it 
![image](https://github.com/fictadvisor/fictadvisor-web/assets/102873980/ef93bf62-8d61-42a1-92d1-3b17b490c156)